airbyte-workers/src/main/java/io/airbyte/workers/temporal/sync/SyncWorkflowImpl.java

```diff
@@ -58,7 +59,9 @@ public StandardSyncOutput run(final JobRunConfig jobRunConfig,
               .withCatalog(run.getOutputCatalog())
               .withResourceRequirements(syncInput.getDestinationResourceRequirements());
 
-          normalizationActivity.normalize(jobRunConfig, destinationLauncherConfig, normalizationInput);
+          final StandardNormalizationSummary normalizationSummary =
```

Comment:
Could you also add a test to `JobCreationAndStatusUpdateActivityTest` to make sure that including a normalizationSummary in the standardSyncOutput that gets persisted works as desired?